### DISCUSSION TITLE
Source maps not working while using React

### DISCUSSION THREAD

**Discussion by 55Cancri** [parcel-bundler/parcel#7179] [Q&A]
When an error occurs, it looks like this in the browser:

![image](https://user-images.githubusercontent.com/17993237/138605699-a2e49a28-2daf-429a-9250-47d6d5b9703e.png)

I don't see the exact file name or anything. I also tried adding:
`"targets": {
    "default": {
      "sourceMap": {
        "inline": true,
        "inlineSources": true
      }
    }
}`
to package.json but that did not work.
The parcel command I am using in my package.json is: `parcel serve ./public/index.html`
Please advise.

**Comment by mischnic** [comment]
Can you provide a reproduction?
